src/ddscxx/include/dds/sub/TDataReader.hpp

```diff
@@ -1628,8 +1628,8 @@ class dds::sub::DataReader : public dds::sub::TAnyDataReader< DELEGATE<T> >
      * can be used to indicate that the created entity should not respond to any of its available
      * statuses. The DDS will therefore attempt to propagate these statuses to its factory.
      * The special constant dds::core::status::StatusMask::all() can be used to select all applicable
-     * statuses specified in the “Data Distribution Service for Real-time Systems Version
-     * 1.2” specification which are applicable to the PublisherListener.
+     * statuses specified in the "Data Distribution Service for Real-time Systems Version
+     * 1.2�" specification which are applicable to the PublisherListener.
```

Comment:
maybe a typo here?

Reply:
You are right: small typo. The previous version was using double quotes that weren't ASCII, which looks nice on webbrowsers,but not in plain code. When replacing them with plain ASCII double quotes, some weird character must have stayed behind. I will correct  this.